### PR TITLE
platform --> .net implementation

### PR DESCRIPTION
**Pull request by tdykstra**
For background on this see https://docs.microsoft.com/en-us/dotnet/standard/glossary#platform

### CODE REVIEW

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/tdykstra-patch-2/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C30%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201708301648575174-464%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: entity-framework/core/platforms/index.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
 ---
-title: EF Core | Platform Support | Microsoft Docs
+title: EF Core | .NET implementations support | Microsoft Docs
```
Comment (divega) [inline entity-framework/core/platforms/index.md]: Supported .NET Implementations?

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/tdykstra-patch-2/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C30%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201708301736447040-464%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by tdykstra** [comment]
We could fix the file name, but I was thinking that's lower priority.  We can change the name and add a redirect from the old name to the new one.

**Pull request merged**